src/client/rooms/room-reducer.js

```diff
@@ -0,0 +1,23 @@
+import { UPDATE_ROOM_USERS, GET_ROOM_DATA } from './room-actions';
+
+const initialState = {
+	users: [],
+	data: {}
+};
+
+const room = (state = initialState, action) => {
+    switch (action.type) {
+        case UPDATE_ROOM_USERS: 
```

Comment:
UPDATE_USER_LIST? This would cause a clash in string consts, but you can namespace UPDATE_USER_LIST for rooms to be `const UPDATE_USER_LIST = 'rooms/UPDATE_USER_LIST;'` and in lobby to use `lobby/` as the prefix